### ISSUE TITLE
aws_db_instance: skip_final_snapshot should be false by default

### ISSUE THREAD

**Issue by aerickson** [hashicorp/terraform#11448]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
Terraform v0.8.4

### Affected Resource(s)
Please list the resources as a list, for example:
- aws_db_instance

### Terraform Configuration Files
```hcl
resource "aws_db_instance" "db" {
  // skip_final_snapshot = true by default
}
```

### Expected Behavior
`skip_final_snapshot = false` by default

The AWS Console's default is to not skip the final snapshot. I think TF should follow AWS defaults (I guess I agree with security deviations from AWS like with SG egress rules).

### Actual Behavior
`skip_final_snapshot = true`

### Steps to Reproduce
It's part of the documented and tested behavior.

### Notes

This caused me data loss. I assume that it will for many others also.

**Comment by stack72** [comment]
Hi @aerickson 

Thanks for coming back to this - I agree with you on this. I think this is something we should do. I don't think we should rush this into a point release as this is backwards incompatible and may actually mean our users incur an extra cost going forward for their resources

I hope you can wait a little while longer for this - please note though, it *will* definitely be part of 0.9 and i will be working on it this week

Thanks

Paul

**Comment by stack72** [comment]
Closed via #11668
